packages/ListBox/src/ListBox.js

```diff
@@ -0,0 +1,161 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Box from "./components/Box";
+import Content from "./components/Content";
+import List from "./components/List";
+import NoResults from "./components/NoResults";
+import Options from "./components/Options";
+import Popover from "./components/Popover";
+import Trigger from "./components/Trigger";
+import useListBox from "./useListBox";
+import handleImperative from "./imperative";
+import {
+  useAdjustWidth,
+  useChildrenLengthChange,
+  useHasFooter,
+  useIsDisabled,
+  useIsPopOverOpen,
+  useOnScrolled,
+  useOptionSelected,
+} from "./hooks";
+
+export const propTypes = {
+  /** Child of type <ListBox.Option /> */
+  children: PropTypes.node,
+
+  /** Disable the entire ListBox */
+  isDisabled: PropTypes.bool,
+
+  /** Let the user to select multiple options at same time */
+  isMulti: PropTypes.bool,
+
+  /** This options will display the listbox without the Popover */
+  isInline: PropTypes.bool,
+
+  /** Indicates if the popover is visible */
+  isOpen: PropTypes.bool,
+
+  /** Callback returning the current selection on the ListBox */
+  onChange: PropTypes.func,
+
+  /** Defaults label to display when the ListBox has not option selected */
+  placeholder: PropTypes.string,
+
+  /** Indicate which is the height for the options container */
+  height: PropTypes.number,
+};
+
+export const defaultProps = {
+  children: null,
+  filter: null, // eslint-disable-line
+  footer: null, // eslint-disable-line
+  height: 200,
+  isDisabled: false,
+  isInline: false,
+  isMulti: false,
+  isOpen: null,
+  onChange: () => {},
+  placeholder: "Select...",
+  trigger: null, // eslint-disable-line
```

Comment:
So `filter`, `footer`, and `trigger` are not in `PropTypes` so that they won't be included in the documentation (via `react-docgen`)? Is that why?

Reply:
exactly they are internals, so I'm trying to avoid to publish them.